### ISSUE TITLE
add in broad specificities for serology labels with only specific specificities to new training/validation sets

### ISSUE THREAD

**Issue by GiovanniBiagini** [lgragert/nn-sero-pytorch#18]

**Comment by GiovanniBiagini** [comment]
Completed by the ser_parse.py script.
